pkgs/tools/graphics/lsix/default.nix

```diff
@@ -20,18 +20,21 @@ stdenvNoCC.mkDerivation rec {
 
     runHook postInstall
   '';
+  wrapperPath = lib.makeBinPath ([
+    (imagemagick.override { ghostscriptSupport = true;})
+  ]);
```

Comment:
This doesn't need to be an a parameter for `mkDerivation`, right? Either just `let` this in `postFixup` or interpolate the `makeBinPath` call directly into the string as before.

Reply:
This allows it to be overwritten by the finalAttrs pattern

Reply:
`postFixup` can be overridden as well, I'd rather not use bespoke patterns without a solid reason